typings/lodash.topath/index.d.ts

```diff
@@ -17,9 +17,8 @@
  * under the License.
  */
 
-export {
-  getTypes,
-  getProperty,
-  getRootProperties,
-  getRootPropertiesObjects,
-} from './lib';
+declare function toPath(value: string | string[]): string[]
```

Comment:
Does this create a `toPath` global? I think it can/should go inside the namespace, no?